common/api/core-backend.api.md

```diff
@@ -346,16 +346,17 @@ export interface AzureBlobStorageCredentials {
     baseUrl?: string;
 }
 
-// @internal
+// @public
 export interface BackendHubAccess {
+    // @internal
     acquireLocks: (arg: BriefcaseDbArg, locks: LockMap) => Promise<void>;
     acquireNewBriefcaseId: (arg: AcquireNewBriefcaseIdArg) => Promise<BriefcaseId>;
     createNewIModel: (arg: CreateNewIModelProps) => Promise<GuidString>;
     deleteIModel: (arg: IModelIdArg & ITwinIdArg) => Promise<void>;
     downloadChangeset: (arg: DownloadChangesetArg) => Promise<ChangesetFileProps>;
```

Comment:
@GytisCepk I ran extract-api and it threw warnings for 2 methods, saying it uses args (`DownloadChangesetArg` and `DownloadChangesetRangeArg`) that are marked as @beta for this API I just converted to @public...  [git blame](https://github.com/iTwin/itwinjs-core/commit/b7192169590e47cd7e09db972b24bdc73b104800#diff-296ede995d70f162e9ac5594f3a209a60cb11782c6e46f971121878a551d363f) showed you most recently worked on this, do you think those args are okay to move to public or should we move `BackendHubAccess` to @beta instead

Reply:
I expect `DownloadChangesetRangeArg` to break due to https://github.com/iTwin/itwinjs-core/issues/7357. So we might mark it `@public` to just deprecate it soon after.

That being said, if there is no other concerns about making `BackendHubAccess` public, I would go ahead and mark both `DownloadChangesetArg` and `DownloadChangesetRangeArg` public.